CMakeLists.txt

```diff
@@ -304,6 +304,7 @@ if (BUILD_SHARED_LIBS)
 endif ()
 
 add_library (${fftw3_lib} ${SOURCEFILES})
+target_include_directories (${fftw3_lib} INTERFACE $<INSTALL_INTERFACE:include>)
```

Comment:
Should there also be a `$<BUILD_INTERFACE:api>`?

Reply:
It would make sense, but as I found, it is not needed. I tried now to use FFTW from the build folder instead of the install folder and the code compiled even without `${BUILD_INTERFACE:api}`.